.github/workflows/pr-container-build.yaml

```diff
@@ -23,9 +23,21 @@ on:
   pull_request_target:
     types: [opened, synchronize, reopened, ready_for_review]
     branches: 
-    - main
-    - rhdh-1.[0-9]+
-    - 1.[0-9]+.x
+      - main
+      - rhdh-1.[0-9]+
+      - 1.[0-9]+.x
+    paths:
+      - '**/docker/**'
+      - '!**/.rhdh/**' # don't build for .rhdh folder changes
+      - '**/.dockerignore'
+      - '**/*.go'
+      - '!**/*_test.go' # exclude test file changes
+      - '**/go.mod'
+      - '**/go.sum'
+      - '**/Makefile'
+      - '**/workflows/*-container-build.yaml'
+      - '**/bundle/**'
+      - '**/config/**'
```

Comment:
Looks good
Not 100% sure but I think config/manager/** would be enough for image build

Reply:
could you test and confirm? or suggest the one-line change? :D